app/src/main/res/values/strings.xml

```diff
@@ -303,22 +313,10 @@
     <string name="preference_follow_device_theme">Follow device theme</string>
 
     <!-- Quick Action Sheet -->
-    <!-- Button in the browser chrome to share the current page -->
-    <string name="quick_action_share">Share</string>
     <!-- Button in the browser chrome to download the current page -->
     <string name="quick_action_download">Download</string>
-    <!-- Button in the browser chrome in the browser to bookmark the current page -->
-    <string name="quick_action_bookmark">Bookmark</string>
-    <!-- Button in the browser chrome in the browser to edit the bookmark of the current page -->
-    <string name="quick_action_bookmark_edit">Edit Bookmark</string>
-    <!-- Button in the browser chrome in the browser to put the the current page in reader mode -->
-    <string name="quick_action_read">Read</string>
     <!-- Button in the browser chrome to exit reader mode -->
     <string name="quick_action_read_close">Close</string>
-    <!-- Button in the browser chrome to open the current page in an external app -->
-    <string name="quick_action_open_app_link">Open in App</string>
-    <!-- Button in the browser chrome to configure reader mode appearance e.g. the used font type and size -->
-    <string name="quick_action_read_appearance">Appearance</string>
     <!-- Content description (not visible, for screen readers etc.): Quick action sheet handle to provide users
         with shortcuts to commonly used actions such as Share, Bookmark etc.. -->
     <string name="quick_action_sheet_handle">Quick actions</string>
```

Comment:
Nit: we can probably remove `quick_action_sheet_handle`, `quick_action_read_close`, and `quick_action_download` strings.

Reply:
Whoops.  I shouldn't have missed that.